scripts/sysdig-probe-loader

```diff
@@ -283,16 +283,17 @@ load_bpf_probe() {
 ARCH=$(uname -m)
 KERNEL_RELEASE=$(uname -r)
 SCRIPT_NAME=$(basename "${0}")
-SYSDIG_PROBE_URL=${SYSDIG_PROBE_URL:-https://s3.amazonaws.com/download.draios.com}
+CONFIG_FILE="/etc/sysdig/${SCRIPT_NAME}"
+if [[ -f "${CONFIG_FILE}" ]]; then
+        source "${CONFIG_FILE}"
+fi
+: ${SYSDIG_PROBE_URL:=https://s3.amazonaws.com/download.draios.com}
```

Comment:
Since this code is changing, we have standardized on https://download.sysdig.com/ instead of the amazonws.com url

Reply:
@mbreitung I'm not sure what you mean; would you like me to change that url into download.sysdig.com?

Reply:
yes, please

Reply:
@gnosek @mbreitung I changed the URL